docs/integration_tests.md

```diff
@@ -0,0 +1,113 @@
+# Running integration tests manually
+
+*This documentation is for developers.*
+
+The integration tests ensure HADDOCK3 workflows work correctly by
+performing short docking or analysis workflows on testing systems. With
+these tests, we evaluate if the final scoring values in the developing
+branch are equal to those obtained in the `main` branch.
+
+You can run the integration tests manually using our helper scripts and
+following the guidelines here.
+
+For a pull request to be accepted, it must pass these integration tests. So we
+kindly ask you to run them before submitting a pull request. We are currently
+working on incorporating these tests into our automatic CI pipelines.
+
+Running the integration tests for a single branch takes about 1 hour using 8 CPU
+cores in a AMD EPYC 7451 24-Core Processor. In case you do not have the computer
+power to run these tests, ask us and we will run them for you.
+
+**How to correctly set up and run the integration tests?**
+
+The easiest way is to have two installations of haddock3:
+one fixed at the `main` branch and other with your developments.
+
+## 1. Install HADDOCK3 main branch separately
+
+First, you should install HADDOCK3 main branch in a separate folder and
+python environment:
+
+```bash
+git clone --recursive https://github.com/haddocking/haddock3 haddock3main
+cd haddock3main
+```
+
+Rename the conda env name in requirements.yml file
+
+```
+sed -i 's/haddock3/haddock3main/g' requirements.yml
+```
+
+Proceed to install the HADDOCK3 main following the installation
+instructions in the repository.
+
+This repository and environment will be the ones you will use to run the
+test examples from the *main* branch.
+
+Once installed, you don't need to install this "main" repository again. You can
+keep using it to test all your developments.
+
+## 3. Running the tests in the "main"
+
+Before running the integration tests, ensure the reference repository (that in
+the "main" branch) is up-to-date.
+
+```bash
+git pull
+```
+
+Remember to activate the corresponding python environment.
+
+```bash
+conda activate haddock3main
+```
+
+Or, if you used `venv`:
+
+```bash
+source venv/bin/activate
+```
+
+Navigate to the `examples/` folder and execute the tests:
+
+```bash
+python run_tests.py -b
+```
+
+This script will run all the test cases in the `examples/` directory and
+stop if any of the tests break. For help, see `python
+run_tests.py -h`.
+
+## 4. Running the tests on your branch
```

Comment:
I use slurm batch submission to run the `*-test.cfg`, and I have the `job` files for that in my account. Initially, I had an explanation for using Slurm, but removed it because I agreed with @rvhonorato comments (https://github.com/haddocking/haddock3/pull/396#discussion_r849123487). But I can add a comment for that now. Should I?

Reply:
Added short message in the docs